templates/CreateNativeTxtFiles.ps1

```diff
@@ -0,0 +1,60 @@
+﻿<#
+.SYNOPSIS
+Export all "export functions" from a native Win32 dll.
+.DESCRIPTION
+TODO
+.PARAMETER AssemblyPath
+The path, including the file name, of the Win32 dll from which the "export functions" name will be exported.
+.PARAMETER OutputPath
+The path where the result file (LIBNAME.exports.txt) will be created.
+#>
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$AssemblyPath,
+    [Parameter()]
+    [string]$OutputPath='.'
```

Comment:
We may want to call this `OutputDir` as that will avoid the question in people's minds of whether the path includes the filename or not.